Documentation/network/bgp-control-plane/bgp-control-plane-troubleshooting.rst

```diff
@@ -95,9 +95,51 @@ Additional new CiliumBGPClusterConfig is not working
 Like the ``CiliumBGPPeeringPolicy``, multiple ``CiliumBGPClusterConfig``
 can select the same node based on the ``nodeSelector`` field. If this is the case,
 the Cilium operator will reject any additional ``CiliumBGPClusterConfig`` from creating
-the ``CiliumBGPNodeConfig`` resource. Following log will be generated by Cilium
-operator indicating this:
-
-.. code:: bash
-
-    level=error msg="skipping node bgpv2-cplane-dev-service-control-plane" error="BGPResourceManager node config bgpv2-cplane-dev-service-control-plane already exist" subsys=bgp-cp-operator
+the ``CiliumBGPNodeConfig`` resource. Following status condition will be set on the
+``CiliumBGPClusterConfig`` to indicate this:
+
+.. code:: yaml
+
+  status:
+    conditions:
+    - lastTransitionTime: "2024-10-26T06:18:04Z"
+      message: 'Selecting the same node(s) with ClusterConfig(s): [cilium-bgp-0]'
+      observedGeneration: 1
+      reason: ConflictingClusterConfigs
+      status: "True"
+      type: cilium.io/ConflictingClusterConfig
+
+Another possibility is that the ``nodeSelector`` in the
```

Comment:
I think this section (NoMatchingNode) would better fit to ~ line 45 where we ask the user to check if node is correctly selected by the nodeSelector.